activesupport/lib/active_support/core_ext/range/compare_range.rb

```diff
@@ -52,29 +52,12 @@ def include?(value)
       end
     end
 
-    # Extends the default Range#cover? to support range comparisons.
-    #  (1..5).cover?(1..5)  # => true
-    #  (1..5).cover?(2..3)  # => true
-    #  (1..5).cover?(1...6) # => true
-    #  (1..5).cover?(2..6)  # => false
-    #
-    # The native Range#cover? behavior is untouched.
-    #  ('a'..'f').cover?('c') # => true
-    #  (5..9).cover?(11) # => false
-    #
-    # The given range must be fully bounded, with both start and end.
-    def cover?(value)
-      if value.is_a?(::Range)
-        is_backwards_op = value.exclude_end? ? :>= : :>
-        return false if value.begin && value.end && value.begin.public_send(is_backwards_op, value.end)
-        # 1...10 covers 1..9 but it does not cover 1..10.
-        # 1..10 covers 1...11 but it does not cover 1...12.
-        operator = exclude_end? && !value.exclude_end? ? :< : :<=
-        value_max = !exclude_end? && value.exclude_end? ? value.max : value.last
-        super(value.first) && (self.end.nil? || value_max.public_send(operator, last))
-      else
-        super
-      end
+    def cover?(value) #:nodoc:
+      ActiveSupport::Deprecation.warn(<<~EOM)
+        ActiveSupport's implementation of Range#cover? is deprecated because the
+        method is part of the Ruby standard library since 2.6.
+      EOM
+      super
```

Comment:
Found a similar commit where a method was replaced by Its native ruby version. See https://github.com/rails/rails/commit/d5197d59a112581e8838f6867c0a02806bace46a 

Hope it helps!

Reply:
Thanks! I've removed the method altogether based on that commit. Although I left the tests to verify existing functionality.